pkg/cli/jobtemplate/describe.go

```diff
@@ -66,15 +67,28 @@ func DescribeJobTemplate(ctx context.Context) error {
 		if err != nil {
 			return err
 		}
-		for _, jobTemplate := range jobTemplates.Items {
+		for i, jobTemplate := range jobTemplates.Items {
+			// Remove managedFields
+			jobTemplate.ManagedFields = nil
 			PrintJobTemplateDetail(&jobTemplate, describeJobTemplateFlags.Format)
+			// Print space if it's not the last element
+			if len(jobTemplates.Items) != 1 && i < len(jobTemplates.Items)-1 {
+				fmt.Printf("\n\n")
+			}
 		}
 		// Get job template detail
 	} else {
 		jobTemplate, err := jobTemplateClient.FlowV1alpha1().JobTemplates(describeJobTemplateFlags.Namespace).Get(ctx, describeJobTemplateFlags.Name, metav1.GetOptions{})
 		if err != nil {
 			return err
 		}
+		// Remove managedFields
+		jobTemplate.ManagedFields = nil
+		// Set APIVersion and Kind if not set
+		if jobTemplate.APIVersion == "" || jobTemplate.Kind == "" {
```

Comment:
Under what circumstance the APIVerision and Kind is empty?

Reply:
refer: https://github.com/k8s-club/k8s-club/issues/14#issuecomment-1555442487
The Chinese discussion here has detailed reasons and source code